packages/requisitions/src/RequestRequisition/DetailView/SidePanel.tsx

```diff
@@ -49,11 +54,64 @@ const AdditionalInfoSection: FC = () => {
   );
 };
 
+const RelatedDocumentsRow: FC<{
```

Comment:
This isn'tvery tidy but I thought to get all the links through before refactoring